### PR TITLE
42 add prop all child variables

### LINKED ISSUE #42: Implement Zeebe propagateAllChildVariables attribute as toggle
__What should we do?__

Implement Zeebe propagateAllChildVariables attribute as toggle
Screenshot_20210602_105608

_Concrete task:_

- [ ] Implement the toggle for propagateAllChildVariables
- [ ] Put it in a new section `Output Propagation`

_Out of scope:_

* Validation

__Why should we do it?__

To migrate zeebe to the new properties panel

__Context__
Child of #9 
Cf. https://github.com/camunda/camunda-modeler-design/issues/19

### PR DESCRIPTION
**Pull request by MaxTru**
closes #42 

propAllChildVars
Screenshot_20210602_155426

### CODE REVIEW

**Review by andreasgeier — COMMENTED** [review]
I know we agreed to have a design finishing round but this toggle is just huge. So I would like to ask for a quick fix to reduce the size to 32x16 px with a 12x12 px circle inside. Please ensure that the label is still aligned after the change.

Expected size of 32x16 px:

Screen Shot 2021-06-03 at 11 40 10

Animation is already included. I'm okay with it.

File: src/bpmn-properties-panel/provider/zeebe/properties/OutputPropagationProps.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+  const setValue = (value) => {
+    const commands = [];
+
+    const businessObject = getBusinessObject(element);
+
+    // (1) ensure extension elements
+    let extensionElements = getExtensionElements(businessObject, 'zeebe:CalledElement');
+
+    if (extensionElements.length === 0) {
```
[the remark this answers is not in the document]
Comment (MaxTru) [reply src/bpmn-properties-panel/provider/zeebe/properties/OutputPropagationProps.js:69]: Done (& testCases added & getExtensionElementsUtils clarified)

**Comment by MaxTru** [comment]
> I know we agreed to have a design finishing round later but this toggle is just huge. So I would like to ask for a quick fix to reduce the size to 32x16 px with a 12x12 px circle inside. Please ensure that the label is still aligned after the change.
> 
> Expected size of 32x16 px:
> 
> Screen Shot 2021-06-03 at 11 40 10
> 
> Animation is already included. I'm okay with it.

Done. I agree that it looks much better
slider

**Review by pinussilvestrus — APPROVED** [review]
Works as expected now 👍  Thanks for the clarification / refactoring regarding `getExtensionElementsList`. 

Still, we have to keep in mind that we have to update the integration tests for the modeling behaviors inside `camunda-bpmn-js` (cf. https://github.com/bpmn-io/properties-panel-next/pull/46#pullrequestreview-674940371). But I don't think we need an issue for that. They should be red once we integrate the new properties.

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; Much better now - thanks!

**Pull request merged**